src/overlays/actors/ovl_En_Ge3/z_en_ge3.c

```diff
@@ -237,6 +237,9 @@ s32 EnGe3_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* p
         // Turn head
         case GELDB_LIMB_HEAD:
             rot->x += this->headRot.y;
+#if PLATFORM_N64
+            break;
+#else
             FALLTHROUGH;
         // This is a hack to fix the color-changing clothes this Gerudo has on N64 versions
         default:
```

Comment:
Wouldn't be easier to ifdef the contents of the `default` case instead of adding a `break` up there?

Reply:
so don't ifdef the fallthrough?

Reply:
yeah, something like
```c
        case GELDB_LIMB_HEAD:
            rot->x += this->headRot.y;

            FALLTHROUGH;
        default:
#if PLATFORM_GC
            // This is a hack to fix the color-changing clothes this Gerudo has on N64 versions
            // cod
#endif
            break;
```

Reply:
I don't like it because obviously N64 versions didn't have
```c
        case GELDB_LIMB_HEAD:
            rot->x += this->headRot.y;

            FALLTHROUGH;
        default:
            break;
```

this is marginally better:

```c
        case GELDB_LIMB_HEAD:
            rot->x += this->headRot.y;
#if PLATFORM_GC
            FALLTHROUGH;
        default:
            // cod
#endif
            break;
```

but I don't like that the break is so far from the case (same as the other suggestion)

Reply:
I know the original probably didn't had the empty `default: break;`, but it is a way to minimize the ifdef'd out code between versions (assuming it matches ofc).

Also to me my suggestion is less confusing to read. It's fine either way I guess

Reply:
Fig points out this is actually a PAL 1.1 change. In light of that I have a slight preference for

```c
        case GELDB_LIMB_HEAD:
            rot->x += this->headRot.y;
#if OOT_VERSION >= PAL_1_1
            FALLTHROUGH;
        default:
            // cod
#endif
            break;
```
over
```c
        case GELDB_LIMB_HEAD:
            rot->x += this->headRot.y;
#if OOT_VERSION < PAL_1_1
            break;
#else
            FALLTHROUGH;
        default:
            // cod
            break;
#endif
```
so you don't have to invert the condition in your head. It probably doesn't matter that much though

Reply:
Changed to cadmic's slight preference
```c
        case GELDB_LIMB_HEAD:
            rot->x += this->headRot.y;
#if OOT_VERSION >= PAL_1_1
            FALLTHROUGH;
        default:
            // cod
#endif
            break;
```